Add Foundation update blog post

clayjohn opened the pull request:
Announce the policies and procedures and provide short updates on other topics of interest.

Tentatively put for Monday, but can easily change the date. 

For socials:
- TL;DR: We've been quiet for awhile, but we have lots of things in the works! It may not be as exciting as shiny new engine features, but we are excited!
- New policies and procedures posted
- Update on hiring
- Continue to expect the asset store
- Working on a proper job board
- Project priorities will be publicly listed
- Trademark policy is in the works

clayjohn committed:
Add Foundation update blog post

Announce the policies and procedures and provide short updates on other topics of interest

Calinou commented on collections/_article/godot-foundation-update-sep-2024.md:
```diff
@@ -0,0 +1,98 @@
+
+## Contractors
+The Foundation has 9 contractors working on the engine directly (5 full time, 4 part time) and 3 contractors
+working on administration + infrastructure right. We are currently in the hiring process for 1-2 more positions.
```
Likely a missing word:

```suggestion
The Foundation has 9 contractors working on the engine directly (5 full time, 4 part time) and 3 contractors
working on administration + infrastructure right now. We are currently in the hiring process for 1-2 more positions.
```

Calinou commented on collections/_article/godot-foundation-update-sep-2024.md:
```diff
@@ -0,0 +1,98 @@
+working on administration + infrastructure right. We are currently in the hiring process for 1-2 more positions.
+
+Even with 12 people we feel very understaffed as there is always more work that can be done. Accordingly, we would like
+to continue hiring once funding allows for it.
```
```suggestion
Even with 12 people, we feel very understaffed as there is always more work that can be done. Accordingly, we would like
to continue hiring once funding allows for it.
```

Calinou commented on collections/_article/godot-foundation-update-sep-2024.md:
```diff
@@ -0,0 +1,98 @@
+### Trademark Policy
+We registered the Godot trademark and the Godot logo trademark while still a member project of the SFC. Last year the
+SFC granted the Godot Foundation ownership over the trademarks. Now we need to have a public policy that says how and
+when others can use the trademarks in order to protect them.
```
"In order" is generally redundant, so it can be removed:

```suggestion
We registered the Godot trademark and the Godot logo trademark while still a member project of the SFC. Last year, the
SFC granted the Godot Foundation ownership over the trademarks. Now we need to have a public policy that says how and
when others can use the trademarks to protect them.
```

Calinou commented on collections/_article/godot-foundation-update-sep-2024.md:
```diff
@@ -0,0 +1,98 @@
+The Foundation (meaning the Board of Directors and the people hired by the Foundation) does not determine the priorities
+of the project in a top-down way. In other words, the Foundation does not dictate to individual volunteers what they
+should work on. Instead it influences the project implicitly by making hiring decisions and hiring people for specific
+tasks that aren’t being done by the community.
```
```suggestion
should work on. Instead, it influences the project implicitly by making hiring decisions and hiring people for specific
tasks that aren’t being done by the community.
```

clayjohn committed:
Apply suggestions from code review

Co-authored-by: Hugo Locurcio <hugo.locurcio@hugo.pro>

clayjohn commented:
@Calinou Thank you! I applied your suggestions

Piralein commented on collections/_article/godot-foundation-update-sep-2024.md:
```diff
@@ -0,0 +1,98 @@
+
+## The Foundation's role in the project
+
+We also wanted to take this chance to muse about about the Foundation's role in the project. In particular, we want to
```
```suggestion
We also wanted to take this chance to muse about the Foundation's role in the project. In particular, we want to
```

mhilbrunner reviewed (approved):
LGTM, besides the typo noted above. Spotted nothing.

BastiaanOlij commented:
Looks really good, my only nitpick is that it's now past September (see title) :)

Thanks Clay!

coppolaemilio committed:
Update collections/_article/godot-foundation-update-sep-2024.md

Co-authored-by: Hana - Piralein <48352564+Piralein@users.noreply.github.com>

JohnVeness commented on collections/_article/godot-foundation-update-sep-2024.md:
```diff
@@ -0,0 +1,98 @@
+
+## Key Policies and Procedures
+The Foundation was formed approximately two years ago, but we have only been operating in a full capacity since last
+summer. We initially adopted the [member project policies](https://sfconservancy.org/projects/policies/) of the Software
```
Probably best to be more explicit than "last summer". Does that mean summer 2023 or summer 2024? And do you mean summer in northern hemisphere?

JohnVeness commented on collections/_article/godot-foundation-update-sep-2024.md:
```diff
@@ -0,0 +1,98 @@
+The Foundation has been working on a number of things behind the scenes that are important for the organization and for
+the Godot ecosystem more broadly. We wanted to take the chance to highlight a few of them in this post.
+
+## Key Policies and Procedures
```
```suggestion
## Key policies and procedures
```

JohnVeness commented on collections/_article/godot-foundation-update-sep-2024.md:
```diff
@@ -0,0 +1,98 @@
+
+## Contractors
+The Foundation has 9 contractors working on the engine directly (5 full time, 4 part time) and 3 contractors
+working on administration + infrastructure right now. We are currently in the hiring process for 1-2 more positions.
```
```suggestion
working on administration and infrastructure right now. We are currently in the hiring process for 1-2 more positions.
```

JohnVeness commented on collections/_article/godot-foundation-update-sep-2024.md:
```diff
@@ -0,0 +1,98 @@
+3. A public project priorities document
+4. A public trademark policy
+
+### Asset Store
```
```suggestion
### Asset store
```

JohnVeness commented on collections/_article/godot-foundation-update-sep-2024.md:
```diff
@@ -0,0 +1,98 @@
+
+We will have a dedicated announcement soon with more information!
+
+### Job Board
```
```suggestion
### Job board
```

JohnVeness commented on collections/_article/godot-foundation-update-sep-2024.md:
```diff
@@ -0,0 +1,98 @@
+contributors to fix high priority bugs that may be impacting your team.
+
+In addition to a job board, we will be curating a list of service providers who have shown that they offer services in
+the Godot Ecosystem to make it easy to find someone to contract with if you need support or any other services.
```
```suggestion
the Godot ecosystem to make it easy to find someone to contract with if you need support or any other services.
```

JohnVeness commented on collections/_article/godot-foundation-update-sep-2024.md:
```diff
@@ -0,0 +1,98 @@
+the Godot Ecosystem to make it easy to find someone to contract with if you need support or any other services.
+
+Much of the foundational work for this has been finished, but it will likely be released after the asset store so we can
+focus our efforts more on the asset store. 
```
```suggestion
focus our efforts on that first. 
```

JohnVeness commented on collections/_article/godot-foundation-update-sep-2024.md:
```diff
@@ -0,0 +1,98 @@
+Much of the foundational work for this has been finished, but it will likely be released after the asset store so we can
+focus our efforts more on the asset store. 
+
+### Project Priorities
```
```suggestion
### Project priorities
```

JohnVeness commented on collections/_article/godot-foundation-update-sep-2024.md:
```diff
@@ -0,0 +1,98 @@
+with expected dates. However, we believe it will answer a lot of questions that the community has about the future of
+the project.
+ 
+### Trademark Policy
```
```suggestion
### Trademark policy
```

clayjohn committed:
Update for December and clean up some language

clayjohn commented:
@coppolaemilio I have updated the doc based on comments here and some of the changes I wanted to do. I also added the mention that the asset store will be ready for testing before the end of the year. 

I set the tentative date for Monday. Feel free to make any adjustments you want

coppolaemilio committed:
Update cover image and small updates
